all_test.go

```diff
@@ -11,79 +11,21 @@ import (
 )
 
 const (
-	clusterPrivateKeyFile string = "cluster_private.key"
-	clusterPublicKeyFile  string = "cluster_public.key"
-	userPrivateKeyFile    string = "user_private.key"
-	usersConfigFile       string = "users.cfg"
-	qdbPort               int    = 30083
-	nodeURI               string = "qdb://127.0.0.1:30083"
-	clusterURI            string = "qdb://127.0.0.1:30083"
-	securedURI            string = "qdb://127.0.0.1:30084"
+	clusterPublicKeyFile string = "cluster_public.key"
+	userPrivateKeyFile   string = "user_private.key"
+	usersConfigFile      string = "users.cfg"
+
+	clusterURI string = "qdb://127.0.0.1:2836"
```

Comment:
Please rename (everwhere / as much as possible): clusterURI -> insecureURI, secured -> secure to make it explicit which cluster is secure and which is insecure.

Reply:
Done.